b2share/modules/users/views.py

```diff
@@ -37,7 +37,7 @@
 blueprint = Blueprint(
     'b2share_users',
     __name__,
-    url_prefix='/users'
+    url_prefix='/user'
```

Comment:
This change breaks the UI, please update this file: https://github.com/EUDAT-B2SHARE/b2share/blob/evolution/webui/src/data/server.js#L14 (just the urls should be enough)